src/components/work-package-details/work-package-details.test.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import { render, screen } from '@testing-library/react';
+import WorkPackageDetails from './work-package-details';
+import { exampleWorkPackage1, exampleWorkPackage2, exampleWorkPackage3, WorkPackage } from 'utils';
+import { dollarsPipe, endDatePipe, fullNamePipe, weeksPipe } from '../../shared/pipes';
+
+describe('Rendering Work Packagae Details Component', () => {
+  test('Rendering example 1', () => {
```

Comment:
just for future reference, you don't need to test 3 different examples if the behavior of the component isn't materially different. an example where you *would* want to test 3 different examples is the change request details component where each different example is materially different because the component conditionally displays different things based on the provided change request object.